alluxiofs/core.py

```diff
@@ -120,7 +120,23 @@ def __init__(
             if preload_path is not None:
                 self.alluxio.load(preload_path)
 
+        def _strip_alluxio_protocol(path):
```

Comment:
can u add doc here?
so this func is to turn
1) alluxio::s3://path  -> into  s3://path
2) s3://path -> into s3://path
and the _strip_protocol is to turn
1) alluxio::s3://path -> path
2) s3://path -> into path
am i correct? 
can u add some doc showing these examples?

Reply:
it only removes alluxio:: from any path